Yoast/wordpress-seo
# Also use replacevar labels in suggestions on Search Appearance settings

## Summary

This PR can be summarized in the following changelog entry:

* Added readable labels to the suggestions in the editors on the Search Appearance pages.

## Relevant technical choices:

* @Xyfi and I had done some diverging work, we decided to go for compatibility rather than rewriting logic for his PR and mine: https://github.com/Yoast/yoast-components/pull/596

## Test instructions

This PR can be tested by following these steps:

* Go to search appearance and verify that replacevariable suggestions there use nicename labels rather than "not-nice" replacevariable tags.

## Quality assurance


Fixes #9997 
Also see: https://github.com/Yoast/yoast-components/issues/577
